libraries/command_line_tools.rb

```diff
@@ -64,7 +64,7 @@ def xcode_version(product)
     end
 
     def macos_version
-      shell_out(['/usr/bin/sw_vers', '-productVersion']).stdout.chomp[/10\.\d+/]
+      shell_out(['/usr/bin/sw_vers', '-productVersion']).stdout.chomp[/1[01]\.\d+/]
```

Comment:
Arguably YAGNI, but it's worth considering implementing the suggested pattern for finding the semantic version from the output of this command. See [Is there a suggested regular expression (RegEx) to check a SemVer string?](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string)

Reply:
That's a good point - arguably this belongs in a more generic library like `MacOS::System` anyway. We'll make that move / improvement when we clean up that library in the `keep_awake` removal.